static/templates/settings/admin-realm-dropdown-stream-list.handlebars

```diff
@@ -0,0 +1,3 @@
+{{#with stream}}
+<li title="{{name}} has {{subscriber_count}} subscribers" class="stream_name" data-stream-id="{{stream_id}}"><a tabindex="-1">{{name}}</a></li>
```

Comment:
can you line-wrap this more nicely?

We can probably remove the name from the title. and just say `{{subscriber_count}} subscribers`.

Well, except we should also tag the title for translation.